_posts/pl/2024-06-14-czy-wiesz-czym-jest-i-jak-dziala-scroll-snap.md

```diff
@@ -0,0 +1,36 @@
+---
+layout:    post
+title:     Czy wiesz, czym jest i jak działa Scroll Snap?
+description: Scroll-snap-type to atrybut CSS, który podczas scrollowania powoduje, że scroll nie zatrzymuje się w miejscu gdzie przestaliśmy scrollować, a przylega do elementu, który mu wskażemy.
+date:      2024-06-14T08:00:00+01:00
+published: true
+didyouknow: true
+lang: pl
+author: pgrobelny
+image: /assets/img/posts/2024-06-14-czy-wiesz-czym-jest-i-jak-dziala-scroll-snap/mouse.webp
+tags:
+- css
+---
+
+`Scroll-snap-type` to atrybut CSS, który podczas scrollowania powoduje, że scroll nie zatrzymuje się w miejscu gdzie przestaliśmy scrollować, a przylega do elementu, który mu wskażemy.
```

Comment:
Scroll-snap-type to atrybut CSS, który powoduje, że scroll nie zatrzymuje się w miejscu, gdzie przestaliśmy scrollować, a przylega do elementu, który mu wskażemy.